packages/firestore/src/local/remote_document_cache.ts

```diff
@@ -95,6 +95,15 @@ export interface RemoteDocumentCache {
     readTime: SnapshotVersion;
   }>;
 
+  /**
+   * Returns the read time of the most recently read document in the cache, or
+   * SnapshotVersion.MIN if not available.
+   */
+  // PORTING NOTE: This is only used for multi-tab synchronization.
+  getLastReadTime(
```

Comment:
Question: why is this declaration here, whereas previously `getLastDocumentChange` wasn't?

Reply:
It allows me to use this method in Persistence without checking `instanceof IndexedDbRemoteDocumentCache`